example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,65 +1,65 @@
-        android:name=".MyApplication"
-        android:icon="@mipmap/ic_launcher">
+    <application
+        android:name="com.clevertap.signedcall_flutter_example.MyApplication"
+        android:icon="@mipmap/ic_launcher"
+        android:label="SC Flutter">
         <activity
-            android:name=".MainActivity"
+            android:name="com.clevertap.signedcall_flutter_example.MainActivity"
+            android:configChanges="orientation|keyboardHidden|keyboard|screenSize|smallestScreenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode"
             android:exported="true"
+            android:hardwareAccelerated="true"
             android:launchMode="singleTop"
             android:theme="@style/LaunchTheme"
-            android:configChanges="orientation|keyboardHidden|keyboard|screenSize|smallestScreenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode"
-            android:hardwareAccelerated="true"
             android:windowSoftInputMode="adjustResize">
             <!-- Specifies an Android theme to apply to this Activity as soon as
                  the Android process has started. This theme is visible to the user
                  while the Flutter UI initializes. After that, this theme continues
                  to determine the Window background behind the Flutter UI. -->
             <meta-data
-              android:name="io.flutter.embedding.android.NormalTheme"
-              android:resource="@style/NormalTheme"
-              />
+                android:name="io.flutter.embedding.android.NormalTheme"
+                android:resource="@style/NormalTheme" />
             <intent-filter>
-                <action android:name="android.intent.action.MAIN"/>
-                <category android:name="android.intent.category.LAUNCHER"/>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
 
-       <service android:name=".MyFirebaseMessagingService"
-           android:exported="false">
-           <intent-filter>
-               <action android:name="com.google.firebase.MESSAGING_EVENT" />
-           </intent-filter>
-       </service>
+        <service
+            android:name="com.clevertap.signedcall_flutter_example.MyFirebaseMessagingService"
+            android:exported="false">
+            <intent-filter>
+                <action android:name="com.google.firebase.MESSAGING_EVENT" />
+            </intent-filter>
+        </service>
 
-       <!--
-       //Out of box solution of CleverTap Android SDK
-       <service android:name="com.clevertap.android.sdk.pushnotification.fcm.FcmMessageListenerService"
-           android:exported="false">
-           <intent-filter>
-               <action android:name="com.google.firebase.MESSAGING_EVENT" />
-           </intent-filter>
-       </service>-->
+        <!--
```

Comment:
@shivamsharma2710 remove this if not needed

Reply:
@darshanclevertap This is resolved!